net/libpsl/Portfile

```diff
@@ -49,7 +49,7 @@ checksums           ${main_distfile} \
 # See e.g. https://trac.macports.org/ticket/60419
 
 # DO NOT change this unless you have understood and acted on the above comment!
-set py_ver          3.10
+set py_ver          3.12
```

Comment:
did you read the comment above and made absolutely sure that nothing would break with this change?